### PR TITLE
1.19.4 random things

### PR DESCRIPTION
**Pull request by eristhea**
me when I test enigma
just trying to improve coverage before 1.19.4

- main goal of this pr is to take messy classes and complete them
- completes mappings in the `text`, `registry`, and `scoreboard` packages
- focuses a lot on the `util` package
- fixes a couple inconsistent names

### CODE REVIEW

File: mappings/net/minecraft/block/DecoratedPotPatterns.mapping [hunk 1]
```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/unmapped/C_hzdaoazh net/minecraft/block/DecoratedPotPatterns
+	FIELD f_auhlbqrf POTTY_PATTERN_SKULL_NAME Ljava/lang/String;
```
Comment (EnnuiL) [inline mappings/net/minecraft/block/DecoratedPotPatterns.mapping]: Potty Pattern? :P
```suggestion
	FIELD f_auhlbqrf POTTERY_PATTERN_SKULL_NAME Ljava/lang/String;
```

File: mappings/net/minecraft/client/gui/DrawableHelper.mapping [hunk 2]
```diff
@@ -3,6 +3,7 @@ CLASS net/minecraft/unmapped/C_smpzexmx net/minecraft/client/gui/DrawableHelper
 		COMMENT The texture of various icons and widgets used for rendering ingame indicators.
 	FIELD f_rlwncfgj STATS_ICON_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
 		COMMENT The texture of icons used in the stats screen.
+	FIELD f_wwyjxrzh LIGHT_DIRT_BACKGROUND_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
```
Comment (EnnuiL) [inline mappings/net/minecraft/client/gui/DrawableHelper.mapping]: oh no; i was so confused by this name but then i realized: it's the actual name of the texture; i don't like it!

File: mappings/net/minecraft/block/DecoratedPotPatterns.mapping [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	FIELD f_cpwmrhfm DECORATED_POT_SIDE_NAME Ljava/lang/String;
+	FIELD f_eomvlzyj POTTERY_PATTERN_ARMS_UP_NAME Ljava/lang/String;
+	FIELD f_iuwqdcay DECORATED_POT_BASE_NAME Ljava/lang/String;
+	FIELD f_jpiddhhm POTTERY_PATTERN_ARCHER_NAME Ljava/lang/String;
+	FIELD f_pkicorcb POTTERY_PATTERN_PRIZE_NAME Ljava/lang/String;
+	FIELD f_tssagzls SHARD_TO_PATTERN Ljava/util/Map;
```
Comment (EnnuiL) [inline mappings/net/minecraft/block/DecoratedPotPatterns.mapping]: hm, if i assume this is what it is, shouldn't it be:
```suggestion
	FIELD f_tssagzls PATTERNS_BY_SHARDS Ljava/util/Map;
```
in order to fit conventions?

**Review by EnnuiL — CHANGES_REQUESTED** [review]
Looks good! Just few things:

File: mappings/net/minecraft/block/DecoratedPotPatterns.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/block/DecoratedPotPatterns.mapping]: :Isa:

Comment (eristhea) [reply mappings/net/minecraft/block/DecoratedPotPatterns.mapping]: oh my gosh I can't even spell iea

**Review by EnnuiL — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Comment by eristhea** [comment]
failing mapping can just be removed

**Pull request merged**